content/en/docs/images/minikf-aws/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+  - cspavlou
+reviewers:
+  - cspavlou
```

Comment:
Let's 1 more approver here (maybe @RFMVasconcelos and/or @alfsuse , for example) and 1-2 more reviewers, so that future PRs don't get stuck. WDYT?